articles/logs/streams/http-event.md

```diff
@@ -0,0 +1,41 @@
+---
+title: HTTP Event Log Streams
+description: HTTP Event Log Streams let you export your events in near real-time to your own server.
+toc: false
+topics:
+ - logs
+ - streams
+ - event-streams
+ - http-event
+contentType: how-to
+---
+
+# HTTP Event Log Streams
+
+HTTP Event Log Streams let you export your log events in near real-time to the server or target of your choice. When Auth0 creates a log entry for your tenant, a copy is sent to a given URL via an HTTP POST request.
+
+If you use Amazon Web Services, Auth0 also offers an [AWS EventBridge integration](/integrations/aws-eventbridge).
+
+## Create an HTTP Event Stream
+
+1. Login to the [Auth0 Dashboard](${manage_url}).
+2. Navigate to **Logs > Streams**.
+3. Click **+ Create Stream**.
+4. Select **Custom Webhook** and enter a unique name for your new HTTP Event Stream.
+5. On the next screen, provide the following settings for your HTTP Event Stream:
+
+| Setting | Description |
+|---------|-------------|
+| Name | A unique display name to distinguish this integration from other integrations |
+| Payload URL | The URL where the event payloads are sent as HTTP POST requests. |
+| Authorization Token | (Optional) Set in the Authorization header of the request if provided. |
+| Content Type | The media type of the payload that will be delivered to the webhook. |
+
+![Create a new HTTP Event Log Stream](/media/articles/logs/http-event-stream.png)
```

Comment:
@erjohnson we've tweaked the UI slightly to improve some copy and indentation of the sample payload. Would it be possible to take a new screenshot?